lib/handler.js

```diff
@@ -1,6 +1,8 @@
-const _ = require('lodash');
-const aeclient = require('aws-elasticsearch-client');
-const AWS = require('aws-sdk');
+const has = require('lodash/has');
+const pick = require('lodash/pick');
+const keys = require('lodash/keys');
```

Comment:
I totally support reducing `lodash` usage and, I think, we can avoid using methods like `has`, `keys`, `defaults` completely.